eWonConnector-gateway/src/main/java/com/hms_networks/SyncManager.java

```diff
@@ -232,41 +232,50 @@ public DeletionResponse process(TagPath t) {
         // Verify integrity of tag history storage information if tag history enabled
         if (historyEnabled && StringUtils.isBlank(tagHistoryStore)) {
             logger.warn(
-                    "History sync is enable, but no history provider has been specified for storage."
-                            + "No data will be stored.");
+                    "History synchronization is enabled, but no history provider " +
```

Comment:
Global comment:

A lot of these multi line strings are changing the existing project format of 

stringpart1 +
(double indent)stringpart2;

Reply:
Your formatter might be getting confused as this project is not the google style standard of 2 space indents so it may not be smart enough to adjust the line wrapping indent based off of the regular indent.  We could do a format commit and change to 2 space if we wanted to change this.

Reply:
I can fix these, or if you want, I can do a formatting commit with Google style once the rest of this PR is reviewed and good to go.